styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesDockerRunner.java

```diff
@@ -302,28 +325,11 @@ public void eventReceived(Action action, Pod pod) {
       }
     }
 
-    private void logEvent(Action action, Pod pod) {
-      final String podName = pod.getMetadata().getName();
-      final String workflowInstance = pod.getMetadata().getAnnotations()
-              .getOrDefault(KubernetesDockerRunner.STYX_WORKFLOW_INSTANCE_ANNOTATION, "N/A");
-      final String status = readStatus(pod);
-      LOG.info("Pod event for {} at resource version {}, action: {}, workflow instance: {}, status: {}",
-               podName, lastResourceVersion, action, workflowInstance, status);
-    }
-
-    private String readStatus(Pod pod) {
-      try {
-        return OBJECT_MAPPER.writeValueAsString(pod.getStatus());
-      } catch (JsonProcessingException e) {
-        return pod.getStatus().toString();
-      }
-    }
-
     private void reconnect() {
       LOG.warn("Re-establishing watching from {}", lastResourceVersion);
 
       try {
-        client.pods()
+        watch = client.pods()
```

Comment:
`watcher.close()` is not very useful. It sends `close` frame to server and print stack trace if `IOException` which it will definitely get. We can skip it.